src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -111,13 +118,21 @@ public void beforeEach() throws Exception {
     StorageTestSuite.deleteAll(itemsStorageUrl(""));
     StorageTestSuite.deleteAll(holdingsStorageUrl(""));
     StorageTestSuite.deleteAll(instancesStorageUrl(""));
+
+    natureOfContentIdsToRemoveAfterTest.clear();
   }
 
   @After
   public void checkIdsAfterEach() {
     StorageTestSuite.checkForMismatchedIDs("instance");
   }
 
+  @After
+  public void removeGeneratedEntities() {
```

Comment:
Did you try deleting all of the nature of content records? Did that cause issues?

Reply:
@marcjohnson-kint I afraid it can cause. Also, we do not have an endpoint to remove all records at once.

Reply:
Hi @marcjohnson-kint, if we clean-up the whole nature-of-content table than we will get this `org.folio.rest.api.ReferenceTablesTest#natureOfContentTermsLoaded` test broken. Not sure if this is a good idea.

Reply:
Hmm, maybe not. This should probably be left out of this work. 

The reference records loading tests should likely be separate to the rest of the tests.